doc/classes/PackedByteArray.xml

```diff
@@ -81,7 +81,7 @@
 			</argument>
 			<description>
 				Returns a new [PackedByteArray] with the data decompressed. Set the compression mode using one of [enum File.CompressionMode]'s constants. [b]This method only accepts gzip and deflate compression modes.[/b]
-				This method is potentially slower than [code]decompress[/code], as it may have to re-allocate it's output buffer multiple times while decompressing, where as [code]decompress[/code] knows it's output buffer size from the beginning.
+				This method is potentially slower than [code]decompress[/code], as it may have to re-allocate it's output buffer multiple times while decompressing, whereas [code]decompress[/code] knows it's output buffer size from the beginning.
```

Comment:
```suggestion
				This method is potentially slower than [code]decompress[/code], as it may have to re-allocate it's output buffer multiple times while decompressing, whereas [code]decompress[/code] knows its output buffer size from the beginning.
```